phpstan.neon

```diff
@@ -3,23 +3,6 @@ parameters:
     paths:
         - src
         - tests
-    ignoreErrors:
```

Comment:
By letting `vendor/bin/phpstan analyse --generate-baseline` generating the errors, they are much more specific and so eventual future inconsistencies emerge earlier